src/status_im/contexts/shell/activity_center/notification/mentions/view.cljs

```diff
@@ -9,25 +9,45 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- mention-text
+  [literal]
+  (let [mention (rf/sub [:messages/resolve-mention literal])]
+    [quo/text
+     {:style style/mention-text
+      :size  :paragraph-1}
+     (str "@" mention)]))
+
+(defn- parsed-text->hiccup
+  [{:keys [literal destination] message-type :type}]
+  (case message-type
+    "mention" [mention-text literal]
+    "link"    destination
+    literal))
```

Comment:
I would it be possible/advisable to convert message-type into a keyword when we decode the message?

Reply:
I think it could be, but we could break existing usages of it. I preferred to keep it as it was.

It isn't normal for us because we usually use keywords, but in this case it isn't too noisy IMO

Reply:
yup yup makes sense 👍